### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
Removed obsolete line, files...and fixed some error in several page.
Please check changes for more detail.

### CODE REVIEW

File: src/components/Login/InputEmailAddr.js [hunk 1]
```diff
@@ -46,15 +45,14 @@ class InputEmailAddr extends Component {
...
-      if (error.code != ErrorMeta.ERR_NONE) {
+      if (error.msg) {
         Alert.alert(
           'Forgot password',
           'Please check email address that you inputted.',
         );
       }
     } else if (result) {
-      Actions.inputSecretCode({
+      Actions.findPassStep2({
```
Comment (SarahhhC) [inline src/components/Login/InputEmailAddr.js]: Need to go back to `inputSecretCode`, it's already refactored.

Comment (MechanicKim) [reply src/components/Login/InputEmailAddr.js]: Yes! She is right!

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ class Login extends Component {
     AsyncStorage.getItem('token', (err, result) => {
       if (err) {
         this.setState({ loaded: true });
-      } else if (err) {
+      } else if (result) {
```
Comment (MechanicKim) [inline src/components/Login/Login.js:35]: Oh! my mistake. Thank you for fixing it

**Review by MechanicKim — COMMENTED** [review]
Thank you for your work!

File: src/components/Login/InputEmailAddr.js (hunk 1 above)
Comment (ifndefdeadmau5) [reply src/components/Login/InputEmailAddr.js]: Okay seems it happened while I resolving merge conflict... I applied wrong version

Comment (MechanicKim) [reply src/components/Login/InputEmailAddr.js]: Thanks trevor!

**Comment by MechanicKim** [comment]
Thanks Trevor. Have a nice week! LGTM

**Pull request merged**